packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -50,6 +51,24 @@ export const PolygonGraph = (props: Props) => {
                         : "var(--movable-line-stroke-weight)",
                 }}
             />
+            {showAngles &&
+                points.map((point, i) => {
+                    const pt1 = points.at(i - 1);
+                    const pt2 = points[(i + 1) % points.length];
+                    if (!pt1 || !pt2) {
+                        return null;
```

Comment:
This return should be unreachable, right? I wonder if we should throw an exception here.

More likely than `pt1` or `pt2` being null is that they might be the same point, which I think will happen if there is only one point in the `points` array. In that case, I think we won't be able to compute an angle, so we probably should return null in that case.

Reply:
This could come in to play in "unlimited" mode.